### DISCUSSION TITLE
How to start tests from the last failed test?

### DISCUSSION THREAD

**Discussion by ghost** [Codeception/Codeception#6649] [Q&A] [answered]
Let's say I run CC with `--fail-fast` option. How do I rerun tests from this point?
Example:
test1 - OK
test2 - OK
test3 - fails, I fix the code and then want to start tests from here (i.e. run tests 3-5) 
test4
test5

**Comment by DavertMik** [comment, marked as answer]
There is an option to run only failed tests

```
./vendor/bin/codecept run -g failed
```

But this will run ONLY failed tests
